src/modules/gimbal/gimbal_params.c

```diff
@@ -146,77 +146,51 @@ PARAM_DEFINE_INT32(MNT_MAN_YAW, 0);
 * @value 0 Disable
 * @value 1 Stabilize all axis
 * @value 2 Stabilize yaw for absolute/lock mode.
-* @min 0
-* @max 2
+* @value 3 Stabilize pitch for absolute/lock mode.
 * @group Mount
 */
 PARAM_DEFINE_INT32(MNT_DO_STAB, 0);
 
 /**
-* Range of pitch channel output in degrees (only in AUX output mode).
+* Max angle of pitch channel output in degrees (only in AUX output mode). Use PWM_CENT for zero-deflection and PWM_MAX for limits.
 *
-* @min 1.0
-* @max 720.0
 * @unit deg
 * @decimal 1
 * @group Mount
 */
-PARAM_DEFINE_FLOAT(MNT_RANGE_PITCH, 90.0f);
+PARAM_DEFINE_FLOAT(MNT_MAX_PITCH, 45.0f);
 
 /**
-* Range of roll channel output in degrees (only in AUX output mode).
+* Min angle of pitch channel output in degrees (only in AUX output mode). Use PWM_CENT for zero-deflection and PWM_MIN for limits.
 *
-* @min 1.0
-* @max 720.0
 * @unit deg
 * @decimal 1
 * @group Mount
 */
-PARAM_DEFINE_FLOAT(MNT_RANGE_ROLL, 90.0f);
+PARAM_DEFINE_FLOAT(MNT_MIN_PITCH, -45.0f);
 
 /**
-* Range of yaw channel output in degrees (only in AUX output mode).
+* Range of roll channel output in degrees (only in AUX output mode). Use PWM_CENT for zero-deflection and PWM_MIN / PWM_MAX for limits.
```

Comment:
It's not really a "range" actually, is it? Rather a "max angle" for both neg and pos? 
I wouldn't adapt the param name but only the description.